pkgs/tools/networking/goflow/default.nix

```
@@ -0,0 +1,27 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "goflow";
+  version = "3.4.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Upxeik2Db7XjPCooCjvBINiAw6Nq+i7oMo9nbG8ecDs=";
+  };
+
+  doCheck = false;
+  vendorSha256 = "sha256-fOlfVI8v7KqNSRhAPlZBSHKfZRlCbCgjnMV/6bsqDhg=";
+
+  meta = with lib; {
+    description = "This application is a NetFlow/IPFIX/sFlow collector in Go.";
```

```suggestion
    description = "A NetFlow/IPFIX/sFlow collector in Go";
```
`This application` is kind of unecessary even if it's in the project's readme. Also [no periods at the end](https://nixos.org/manual/nixpkgs/stable/#var-meta-description).